src/View/Helper/FormHelper.php

```diff
@@ -65,6 +65,16 @@ class FormHelper extends Helper
     protected $_datetimeParts = ['year', 'month', 'day', 'hour', 'minute', 'second', 'meridian'];
 
     /**
+     * Special options used for datetime inputs.
+     *
+     * @var array
+     */
+    protected $_datetimeOptions = [
+        'interval', 'round', 'monthNames', 'minYear', 'maxYear',
```

Comment:
That's what I had started doing earlier but then switched to using a common list at it was more convenient to maintain.